### PR TITLE
feat: log web vitals

### PR DESCRIPTION
**Pull request by reidrankin**
## Description

Don't swallow CRA's `reportWebVitals` output.

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -26,7 +26,8 @@ serviceWorker.unregister()
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals()
+// eslint-disable-next-line no-console
+reportWebVitals(x => console.log('reportWebVitals', x))
```
Comment (cjthompson) [inline src/index.tsx:30]: You can avoid the eslint issue by using `console.info` or `console.debug`.

Comment (reidrankin) [reply src/index.tsx:30]:
```suggestion
reportWebVitals(x => console.debug('reportWebVitals', x))
```
good catch!

Comment (reidrankin) [reply src/index.tsx]: huh, doesn't look like it liked that. still, debug log level is the way to go. (and we can't use process.env.NODE_ENV b/c this is in the browser)

Comment (cjthompson) [reply src/index.tsx]: react adds in process.env as a way to get environment variables that were compiled into the app.

Comment (cjthompson) [reply src/index.tsx]: ok so `debug` still gets the eslint error but `console.info`, `console.warn` and `console.error` do not.

**Review by cjthompson — APPROVED** [review]

**Review by notatooyou — APPROVED** [review]

**Pull request merged**